src/Apps/Legion/Components/WizardShell/Internal/ModifyStep/Internal/TypesTab/TypesTab.styles.ts

```diff
@@ -0,0 +1,21 @@
+import { ITypesTabStyles } from './TypesTab.types';
+import {
+    IProcessedStyleSet,
+    mergeStyleSets,
+    memoizeFunction
+} from '@fluentui/react';
+
+export const getStyles = memoizeFunction(
+    (): IProcessedStyleSet<ITypesTabStyles> => {
+        return mergeStyleSets({
+            root: {
+                maxHeight: 400,
+                overflowY: 'auto',
+                display: 'flex',
+                flexDirection: 'column',
+                rowGap: 8,
+                marginTop: 8
```

Comment:
is `rowGap` how you get the Stack functionality?

Reply:
yes, this is equivalent to the Stack token functionality